platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -533,20 +534,21 @@ check_architecture() {
 
 check_docker_api_version() {
     # Check dependencies
+    wrap_debug_message "Checking docker api client version..."
 
     #TODO : This is how we check  for a container engine in our packages. Is this the right way?
     ret=$(need_cmd docker)
     if [ "$?" -ne 0 ]; then
         wrap_warning "check_docker_api_version"
-        wrap_warning_message "Docker Engine does not exist on this device!!, Please follow instructions here on how to install a compatible container engine
+        wrap_warning_message "Docker Engine does not exist on this device, Please follow instructions here on how to install a compatible container engine
         https://docs.microsoft.com/en-us/azure/iot-edge/how-to-provision-single-device-linux-symmetric?view=iotedge-2020-11&tabs=azure-portal%2Cubuntu#install-a-container-engine"
         return
     fi
 
-    version=$(docker version -f '{{.Client.APIVersion}}')
+    version=$(docker version -f '{{.Client.APIVersion}}'>/dev/null 2>&1)
     if [ $? != 0 ]; then
         wrap_warning "check_docker_api_version"
-        wrap_warning_message "Could not get Docker Version"
+        wrap_warning_message "Could not get Docker Version. Run it with root privileges"
```

Comment:
I think we should remove the root priveleges statement. In the beginning of the script, we already have a check to see if the script is running with root privileges or not.